internal/cmd/code_server_test.go

```diff
@@ -1,3 +1,5 @@
+//go:test !darwin
```

Comment:
Go tired of running into third-party rate-limiting. It's totally fine to not run this test locally as long as it runs in C/I.